virttest/utils_misc.py

```
@@ -4466,6 +4466,102 @@ def get_distro(session=None):
+    """Parse kernel version into major.minor.stable format"""
+    import re
+
+    try:
+        version_match = re.match(r"(\d+)\.(\d+)\.(\d+)", kernel_version)
+        if version_match:
+            return {
+                "major": int(version_match.group(1)),
+                "minor": int(version_match.group(2)),
+                "stable": int(version_match.group(3)),
+            }
+    except:
```

Ruff (0.13.3)

Do not use bare `except`

(E722)